molo/surveys/templatetags/molo_survey_tags.py

```diff
@@ -54,9 +72,15 @@ def surveys_list_headline(context):
 
 
 @register.inclusion_tag('surveys/surveys_list.html', takes_context=True)
-def surveys_list(context, pk=None):
-    context = get_survey_list(context)
-    return add_form_objects_to_surveys(context)
+def surveys_list(context, pk=None, only_linked_surveys=False,
+                 only_direct_surveys=False):
+    context = get_survey_list(context,
+                              only_linked_surveys=only_linked_surveys,
+                              only_direct_surveys=only_direct_surveys)
+    if only_linked_surveys:
```

Comment:
No, the linked surveys don't need the forms to be added, because they are only displayed in the linked page, not on this page. It's a small optimisation.